src/transformers/onnx/config.py

```diff
@@ -195,42 +210,75 @@ def use_external_data_format(num_parameters: int) -> bool:
             >= EXTERNAL_DATA_FORMAT_SIZE_LIMIT
         )
 
+    def _generate_dummy_images(
+        self, batch_size: int = 2, num_channels: int = 3, image_height: int = 40, image_width: int = 40
+    ):
+        images = []
+        for _ in range(batch_size):
+            data = np.random.rand(image_height, image_width, num_channels) * 255
+            images.append(Image.fromarray(data.astype("uint8")).convert("RGB"))
+        return images
+
     def generate_dummy_inputs(
```

Comment:
This base method now has a mix of arguments for text and image modalities. I'm not 100% sure if we should split the modalities apart ...